stlTranslator/SourceFiles/read.cpp

```diff
@@ -10,23 +11,22 @@ Read::Read()
 }
 
 void Read::read() {
-    ifstream myFile("sphere.stl");
-
-    if (myFile.is_open()) {
```

Comment:
you can use error handling if file not get open 
if (!myFile) {  
        cerr << "Error: Could not open " << filename << endl;
        return;
    }